frontend/src/components/features/chat/event-content-helpers/get-action-content.ts

```diff
@@ -111,7 +111,7 @@ export const getActionContent = (event: OpenHandsAction): string => {
       return getIPythonActionContent(event);
     case "browse":
       return getBrowseActionContent(event);
-    case "browse_interactive":
+    case i18n.t("ACTION$BROWSE_INTERACTIVE"):
```

Comment:
```suggestion
    case "browse_interactive":
```

We shouldn't translate the action type here, instead we should add an exception to the script

Reply:
Done!